orientdb/src/main/java/org/hibernate/datastore/ogm/orientdb/logging/impl/Log.java

```diff
@@ -52,4 +43,37 @@
 
 	@Message(id = 1408, value = "Error while cheking transaction status")
 	HibernateException exceptionWhileChekingTransactionStatus(@Cause Exception e);
+
+	@Message(id = 1700, value = "Cannot create class %s")
+	HibernateException cannotGenerateVertexClass(String className, @Cause Exception cause);
```

Comment:
All the id in this class should be in the same range, could you update the existing one as well to 17xx instead of 14xx